pootle/apps/pootle_statistics/migrations/0011_cleanup_submissions_squashed_0023_remove_scorelog.py

```diff
@@ -0,0 +1,175 @@
+        | subs.exclude(creation_time=F("suggestion__review_time"))
+        | subs.exclude(submitter=F("suggestion__reviewer")))
+    to_delete.delete()
+
+
+def convert_accept_suggestion_subs(apps, schema_editor):
+    subs = apps.get_model("pootle_statistics.Submission").objects
+
+    # clear up existing subs first
+    _assoc_accept_suggestion_subs(apps)
+    _delete_redundant_accept_sugg_subs(apps)
+
+    # convert to SubmissionTypes.WEB
+    subs.filter(type=SUGG_ACCEPT_FLAG).update(type=WEB_FLAG)
+
+
+def convert_check_subs(apps, schema_editor):
+    subs = apps.get_model("pootle_statistics.Submission").objects
+    subs.filter(type=MUTE_FLAG).update(
+        field=CHECK_FIELD_FLAG,
+        type=WEB_FLAG,
+        old_value=UNMUTED,
+        new_value=MUTED)
+    subs.filter(type=UNMUTE_FLAG).update(
+        field=CHECK_FIELD_FLAG,
+        type=WEB_FLAG,
+        old_value=MUTED,
+        new_value=UNMUTED)
+
+
+def set_suggestion_submitters(apps, schema_editor):
+    subs = apps.get_model("pootle_statistics.Submission").objects
+    accepted_subs = subs.filter(suggestion__state__name="accepted").values_list(
+        "pk", "suggestion__user_id")
+    for pk, user in accepted_subs.iterator():
+        subs.filter(pk=pk).update(submitter_id=user)
+
+
+class Migration(migrations.Migration):
+
+    replaces = [
+        (b'pootle_statistics', '0011_cleanup_submissions'),
+        (b'pootle_statistics', '0012_remove_create_subs'),
+        (b'pootle_statistics', '0013_remove_revert_flag'),
+        (b'pootle_statistics', '0014_truncate_source_subs'),
+        (b'pootle_statistics', '0015_remove_system_scorelogs'),
+        (b'pootle_statistics', '0016_drop_add_suggestion_subs'),
+        (b'pootle_statistics', '0017_drop_reject_suggestion_subs'),
+        (b'pootle_statistics', '0018_remove_submission_store'),
+        (b'pootle_statistics', '0019_convert_accept_sugg_subs'),
+        (b'pootle_statistics', '0020_convert_checks_subs'),
+        (b'pootle_statistics', '0021_remove_paid_rates'),
+        (b'pootle_statistics', '0022_set_suggestion_submitters'),
+        (b'pootle_statistics', '0023_remove_scorelog')]
+
+    dependencies = [
+        ('pootle_store', '0045_remove_suggestion_tmp_state'),
+        ('pootle_statistics', '0010_submission_on_delete_user')]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='scorelog',
+            unique_together=set([]),
+        ),
+        migrations.RemoveField(
+            model_name='scorelog',
+            name='submission',
+        ),
+        migrations.RemoveField(
+            model_name='scorelog',
+            name='user',
+        ),
+        migrations.RemoveField(
+            model_name='scorelog',
+            name='rate',
+        ),
+        migrations.RemoveField(
+            model_name='scorelog',
+            name='review_rate',
+        ),
+        migrations.DeleteModel(
+            name='ScoreLog',
```

Comment:
I have never tried it, but is it not possible to just drop the `ScoreLog` model instead of deleting fields from it and then dropping the `ScoreLog` model as we are doing here?